ci/docs/build.sh

```diff
@@ -18,33 +18,37 @@ export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
 export PROJECTS=(cudf libcudf)
 
-logger "Check environment..."
+gpuci_logger "Check environment..."
 env
 
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage..."
 nvidia-smi
 
-logger "Activate conda env..."
-source activate rapids
+gpuci_logger "Activate conda env..."
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
 # TODO: Move installs to docs-build-env meta package
-conda install -c anaconda beautifulsoup4 jq
+gpuci_conda_retry install -c anaconda beautifulsoup4 jq
```

Comment:
This whole `TODO` block can probably be removed. I see `sphinx-markdown-tables` and `beautifulsoup4` in the [rapids-doc-env](https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/rapids-doc-env/meta.yaml#L37-L49) and `jq` in our [gpuci devel images](https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.16/rapidsai/devel.Dockerfile#L80-L83).

(GitHub won't let me make this code suggestion since it spans lines of code that you didn't modify)